modules/registry/registry-core/src/main/java/org/apache/airavata/registry/core/entities/expcatalog/ComputationalResourceSchedulingEntity.java

```diff
@@ -0,0 +1,162 @@
+package org.apache.airavata.registry.core.entities.expcatalog;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+/**
+ * Persistent class for computational_resource_scheduling data table.
+ */
+@Entity
+@Table(name = "COMPUTE_RESOURCE_SCHEDULING")
+@IdClass(ComputationalResourceSchedulingPK.class)
+public class ComputationalResourceSchedulingEntity implements Serializable {
```

Comment:
I don't think this is needed. There's a mismatch between the Thrift data model and the database entities here. UserConfigurationDataModel and ComputationalResourceSchedulingModel map to the single UserConfigurationDataEntity/table in the database.

Reply:
@isururanawaka no I'm saying this Entity class isn't needed. These fields from the Thrift model are saved into the user_configuration_data table.